File: app/src/main/java/com/wealoha/social/beans/ProfileViewResult.java
package com.wealoha.social.beans;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.Map;

import com.wealoha.social.beans.feed.Feed;
import com.wealoha.social.beans.imagemap.HasImageMap;

/**
 * Created by walker on 14-3-25.
 */
public class ProfileViewResult extends ResultData implements HasImageMap {

	public String nextCursorId;
	public HashMap<String, Integer> commentCountMap;
	public HashMap<String, Integer> likCountMap;
	public HashMap<String, Object> userMap;
	public ArrayList<Feed> list;
	public Map<String, Image> imageMap;

	@Override
	public Map<String, Image> getImageMap() {
		return imageMap;
	}
}

File: app/src/main/java/com/wealoha/social/beans/ProfileViewResult.kt
package com.wealoha.social.beans

import com.wealoha.social.beans.feed.Feed
import com.wealoha.social.beans.imagemap.HasImageMap
import java.util.*

/**
 * Created by walker on 14-3-25.
 */
class ProfileViewResult : ResultData(), HasImageMap {
    var nextCursorId: String? = null
    var commentCountMap: HashMap<String, Int>? = null
    var likCountMap: HashMap<String, Int>? = null
    var userMap: HashMap<String, Any>? = null
    var list: ArrayList<Feed>? = null
    override var imageMap: Map<String, Image>? = null
}